src/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -4432,7 +4432,7 @@ public void Shutdown(SocketShutdown how)
             GlobalLog.Print("Socket#" + Logging.HashString(this) + "::Shutdown() how:" + how.ToString());
```

Comment:
I'm going to leave that as-is as it's unrelated to this change. Please file an issue for follow-up (or add this to #3441).